app/main.py

```diff
@@ -1 +1,41 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str):
+        self.name: str = name
+        self.health: int = 100
+        self.hidden: bool = False
+        Animal.alive.append(self)
+
+    def __del__(self):
+        if self in Animal.alive:
+            Animal.alive.remove(self)
+
+    def __repr__(self) -> str:
+<<<<<<< HEAD
```

Comment:
what is it?

Reply:
just tried to commit fast until power did`nt switch off, and git showed me mistakes i did by this way